diff --git a/node_modules/eccrypto/browser.js b/node_modules/eccrypto/browser.js
index c9847df..1bd9491 100644
--- a/node_modules/eccrypto/browser.js
+++ b/node_modules/eccrypto/browser.js
@@ -6,7 +6,9 @@ var ec = new EC("secp256k1");
 var browserCrypto = global.crypto || global.msCrypto || {};
 var subtle = browserCrypto.subtle || browserCrypto.webkitSubtle;
 
-var nodeCrypto = require('crypto');
+var nodeCrypto = require('crypto-browserify');
+
+var Buffer = require('buffer/').Buffer
 
 const EC_GROUP_ORDER = Buffer.from('fffffffffffffffffffffffffffffffebaaedce6af48a03bbfd25e8cd0364141', 'hex');
 const ZERO32 = Buffer.alloc(32, 0);
